.github/workflows/CI.yml

```diff
@@ -726,6 +726,25 @@ jobs:
           cd testing-framework/terraform
           make checkCacheHits
 
+  update-ci-status:
+    runs-on: ubuntu-latest
+    needs: [validate-all-tests-pass]
```

Comment:
Should we move this to have a dependency on `release-candidate`? There are still steps that could fail during that job.

Reply:
Good point. moved to depend on the release candidate job.